SU2_CFD/src/solver_structure.cpp

```diff
@@ -226,7 +226,8 @@ void CSolver::SetResidual_RMS(CGeometry *geometry, CConfig *config) {
       
     }
     
-    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/Global_nPointDomain)));
+    SetRes_RMS(iVar, max(EPS*EPS, sqrt(rbuf_residual[iVar]/ geometry->GetnPoint() )));
```

Comment:
I think this will only work when the code is built in serial. This is a special case when nPoint = Global_nPointDomain. Otherwise, we should use MPI to gather the total number of grid nodes in the mesh (Global_nPointDomain above), excluding ghost nodes, so that we are always dividing by the same number of grid nodes regardless of number of MPI ranks.